### PR TITLE
Create get_SWH_equipment_associated_with_each_swh_distribution_system.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

**Review by claperle — APPROVED** [review]

**Review by JacksonJ-KC — APPROVED** [review]

File: docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## get_SWH_equipment_associated_with_each_swh_distribution_system
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md:1]: it's providing a dictionary where the keys are the ids of each SWH distribution system.

File: docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Description: This function gets all the SWH equipment connected to a SWH distribution system.  The information is stored in a dictionary where the keys are the SWH Distribution System Ids and values are a dictionary giving the ServiceWaterHeatingEquipment, and Pumps connected to the particular use type
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md:3]: distribution system.  my bad

File: docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JacksonJ-KC) [reply docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md:1]: OK. updated comment

[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_SWH_equipment_associated_with_each_swh_distribution_system.md]: updated file name

**Pull request merged**